src/main/java/com/bouncestorage/swiftproxy/v1/ContainerResource.java

```diff
@@ -237,6 +240,12 @@ public Response listContainer(@NotNull @PathParam("container") String container,
     @XmlRootElement(name = "object")
     @XmlType
     static class ObjectEntry {
+        static class SwiftDateSerializer extends DateSerializer {
```

Comment:
dates returned from http header has normal date format (which is not serialized by jackson). container listing (GET account) does not have container creation date in the output